sssom/cli.py

```diff
@@ -675,5 +675,26 @@ def annotate(input: str, output: TextIO, replace_multivalued: bool, **kwargs):
     )
 
 
+@main.command()
+@input_argument
+@click.option(
+    "--remove-map",
```

Comment:
This is great, we can later consider making this multivalued (not now). Thank you!